src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,176 @@
+
+To create a flex configuration file follow these steps:
+
+1. Path:
+```
+/etc/newrelic-infra/integrations.d
+```
+
+2. Create a file with the name `lighttpd-http-config.yml`
+
+Note: A generic file name could be &ltFILENAME&gt-config.yml
+
+3. Use add the following configuration to your `lighttpd-http-config.yml` file. 
+
+Sample configuration file: https://github.com/newrelic/nri-flex/blob/master/examples/lighttpd-http-example.yml
+
+It's important to not that you can replace `base_url` and `url` with your application url (Please refer to url documentation for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
+
+Once your configuration file is updated, it will look like this:
+
+```
+---
+integrations:
+  - name: nri-flex
+    # interval: 30s
+    config:
+      name: lighttpdFlex
+      global:
+        base_url: http://lighttpd.net/
+      apis:
+        - event_type: status
+          url: status?format=plain
+        - event_type: &lTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - cache: status?format=plain
+              split_by: ": "
+          snake_to_camel: true
+
+
+  - name: nri-flex
+    interval: 10s
+    # timeout: 5s
+    config:
+      name: test
+      apis:
+        - event_type: &ltTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - run: curl 'http://lighttpd.net/status?auto'
+              split_by: ": "
+              regex_matches:
+                - expression: (\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)
+```
+
+</Step>
+
+<Step>
+
+### Restart the New Relic infra agent and lighttpd service.
+
+To restart the New Relic infra agent follow the instructions in [this doc](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) and run this command:
+
+```
+sudo systemctl restart lighttpd.service && sudo systemctl restart newrelic-infra.service
+```
+</Step>
+
+<Step>
+###  Query your data using NRQL
+
+You can use this sample query to view your Lighttpd metrics captured on [one.newrelic.com](https://one.newrelic.com/all-capabilities):
+
+```
+FROM <EVENT_TYPE> SELECT * LIMIT MAX 
+```
+
+EVENT_TYPE should match with the one you mentioned in the nri-flex config file.
+</Step>
+
+</Steps>
+
+## See Lighttpd metrics in a dashboard [#dash]
+
+With only the infrastructure agent installed and instrumented with your app, you can view your raw data in our Metrics & events. Our default dashboards transform that raw data into charts and graphs, which provide a bird's eye view of your system's health. To install our default dashboards, go to our Lighttpd instant observability page.
```

Comment:
Although it's not yet on the page, I'd add the link to the observability page or you can ask the people involved if they know the URL. I imagine that it can be something like that: "...go to our [Lighttpd instant observability](https://newrelic.com/instant-observability/lighttpd) page."

Reply:
yes! that makes sense! thannks!